stock_inventory_discrepancy/tests/test_inventory_discrepancy.py

```diff
@@ -0,0 +1,228 @@
+            {
+                "name": "Test Manager",
+                "login": "manager",
+                "email": "test.manager@example.com",
+                "groups_id": [(6, 0, [group_stock_man.id, group_inventory_all.id])],
+            }
+        )
+        group_stock_user = self.env.ref("stock.group_stock_user")
+        group_inventory = self.env.ref(
+            "stock_inventory_discrepancy.group_stock_inventory_validation"
+        )
+        self.user = self.env["res.users"].create(
+            {
+                "name": "Test User",
+                "login": "user",
+                "email": "test.user@example.com",
+                "groups_id": [(6, 0, [group_stock_user.id, group_inventory.id])],
+            }
+        )
+
+        self.user_2 = self.env["res.users"].create(
+            {
+                "name": "Test User 2",
+                "login": "user_2",
+                "email": "test2.user@example.com",
+                "groups_id": [(6, 0, [group_stock_user.id, group_inventory_all.id])],
+            }
+        )
+
+        self.no_user = self.env["res.users"].create(
+            {
+                "name": "No User",
+                "login": "no_user",
+                "email": "test.no_user@example.com",
+                "groups_id": [(6, 0, [group_employee.id])],
+            }
+        )
+
+        self.quant_line1 = self.obj_quant.with_context(inventory_mode=True).create(
+            {
+                "product_id": self.product1.id,
+                "quantity": 2.0,
+                "location_id": self.test_loc.id,
+            }
+        )
+        self.quant_line2 = self.obj_quant.with_context(inventory_mode=True).create(
+            {
+                "product_id": self.product2.id,
+                "quantity": 4.0,
+                "location_id": self.test_loc.id,
+            }
+        )
+
+        # starting_inv = self.obj_inventory.create(
+        #     {
+        #         "name": "Starting inventory",
+        #         "line_ids": [
+        #             (
+        #                 0,
+        #                 0,
+        #                 {
+        #                     "product_id": self.product1.id,
+        #                     "product_uom_id": self.env.ref("uom.product_uom_unit").id,
+        #                     "product_qty": 2.0,
+        #                     "location_id": self.test_loc.id,
+        #                 },
+        #             ),
+        #             (
+        #                 0,
+        #                 0,
+        #                 {
+        #                     "product_id": self.product2.id,
+        #                     "product_uom_id": self.env.ref("uom.product_uom_unit").id,
+        #                     "product_qty": 4.0,
+        #                     "location_id": self.test_loc.id,
+        #                 },
+        #             ),
+        #         ],
+        #     }
+        # )
+        # starting_inv.action_force_done()
```

Comment:
Remove commented lines

Reply:
@carolinafernandez-tecnativa please attend this

Reply:
@pedrobaeza @sergio-teruel done!